src/process_viewer.rs

```diff
@@ -112,8 +112,8 @@ impl Procs {
         });
         kill_button.set_sensitive(false);
 
-        vertical_layout.add(&scroll);
-        vertical_layout.add(&kill_button);
+        vertical_layout.pack_start(&scroll, true, true, 0);
+        vertical_layout.pack_start(&kill_button, false, true, 0);
```

Comment:
Before this commit (without my changes):
When you resize processviewer's window larger than it initially is, the GtkScrolledWindow containing all processes won't expand. After applying these changes it will.